documentation/src/main/asciidoc/topics/con_in_memory_caching.adoc

```diff
@@ -0,0 +1,21 @@
+[id='in-memory-caching_{context}']
+= In-memory caching
+
+{brandname} provides applications with data access that has much lower latency for read operations and higher throughput for write operations in comparison with traditional database systems.
+For in-memory caches, {brandname} stores key/value pairs in RAM only.
+
+.In-memory caches
+//Community content
+ifdef::community[]
+image::in-memory-cache-ispn.png[This illustration depicts a {brandname} cluster of four server nodes with in-memory caches that allow applications to perform read and write operations through remote clients.]
```

Comment:
I'm failing to see what this image has to do with in-memory caches since it only shows the client-server architecture.

Reply:
In a separate topic we mention that client/server is preferable to embedded because the server gives you logging and monitoring and separates the cache from application logic. This diagram is trying to be consistent with that message by showing remote caches in a dedicated JVM. So it's still in memory but not in the same memory space as code execution. I'll make some changes to the wording here but do you think this should be "In-memory caches in a client/server architecture" or should those two things be completely distinct?

Reply:
@pruivo Excellent comment, as always. I've actually just dropped the "in-memory" topic. We can include the diagram as part of the deployment model maybe. Or we could drop it altogether if it's not adding value.